src/main/java/intellistart/interviewplanning/model/booking/BookingRepository.kt

```diff
@@ -0,0 +1,70 @@
+package intellistart.interviewplanning.model.booking
```

Comment:
relocate to repository package

Reply:
my package structure is model-oriented but not level-oriented